content/Personal/a_year_at_mozilla.rst

```diff
@@ -0,0 +1,281 @@
+of alternative to Pocket. We succeeded in setting up a stack, deploying and scaling
+it quite rapidly, working sometimes late at night. Since we wanted to capitalize on all the efforts we had done
+(API design, scaling, ...) we created `a toolkit called Cliquet <http://www.servicedenuages.fr/en/why-cliquet>`_.
+I could talk about it at `PyBCN <http://mozilla-services.github.io/cliquet/talks/2015.07.pybcn/>`_
+and `Pycon-FR <http://mozilla-services.github.io/cliquet/talks/2015.10.pyconfr/>`_.
+
+We even hacked on `readability <https://github.com/mozilla/readability>`_, a JavaScript library
+that transforms a webpage into plain text. You probably executed it already if you used the `Firefox Reader Mode <https://support.mozilla.org/en-US/kb/firefox-reader-view-clutter-free-web-pages>`_.
+Before joining our team officially, N1k0 made a `full-Web version of a readinglist app
+<https://github.com/mozilla-services/readinglist-client>`_ :)
+
+Unfortunately, the readinglist project was shutdown at the last moment. That was hard to take,
+and we had quite a hard time recovering from this deception.
+
+
+Booting Kinto
+-------------
+
+We revamped the server code and saw an opportunity to bring back our idea of `a generic remote storage <http://www.servicedenuages.fr/en/generic-storage-ecosystem>`_.
+Our first concrete use-case was a prototype
+for a payment and subscription service, which became a pretext to implement a fine-grained
+permissions API into our newly born Kinto.
+
+::
+
+    > On Fri 26 December 2014, Mathieu wrote:
+    > [...]
+    > Whether Daybed will become a true Mozilla project or not is totally
+    > unclear.
+    >
+
+Daybed never became a Mozilla project, but its concepts were all re-implemented
+in Kinto!
+
+Last summer, Firefox OS had no way to synchronize bookmarks and browsing history
+accross devices. We built `kinto.js <https://github.com/Kinto/kinto.js>`_, a JavaScript
+client for Kinto, focused on offline capabilities and records synchronization.
+Personnally, I learned a lot from this project. N1k0 was leading us towards modern
+and elegant patterns in JavaScript, and we could get our hands dirty with IndexedDB.
+
+As a side project I could spend some time on an offline-first Web app, `Routina <http://github.com/leplatrem/Routina/>`_,
+for which we did `a Kinto+React boilerplate <https://github.com/Kinto/kinto-react-boilerplate/>`_.
+
+We saw an opportunity to spread Kinto within the organization with the `Go Faster initiative
+<http://www.servicedenuages.fr/en/summer-perspectives>`_, which consists in shipping
+settings, content and features quicker that the 6-weeks release cycle.
+
+We even `did some experiments <https://github.com/mozilla-services/servicedenuages.fr/blob/7a3edb700d46b41207a8df97b6341f184818732c/content/2016.02.localisation-service.md>`_
+to have the UI translations shipped asynchronously!
```

Comment:
There is also a little demo video here: https://kinto-ota.dev.mozaws.net/attachments/demo.mp4